pkg/sqlmodel/multirow.go

```diff
@@ -122,6 +122,152 @@ func GenDeleteSQL(changes ...*RowChange) (string, []interface{}) {
 	return buf.String(), args
 }
 
+// GenUpdateSQLFast generates the UPDATE SQL and its arguments.
+// Input `changes` should have same target table and same columns for WHERE
+// (typically same PK/NOT NULL UK), otherwise the behaviour is undefined.
+// It is a faster version compared with GenUpdateSQL.
+func GenUpdateSQLFast(changes ...*RowChange) (string, []interface{}) {
```

Comment:
Should the batch size be limited ?

Reply:
I have summited https://github.com/pingcap/tiflow/issues/8084, and this limit will be added in another PR.